src/tango/xdp/fd_xdp_ctl_help

```diff
@@ -0,0 +1,117 @@
+
+Usage: %s [cmd] [cmd args] [cmd] [cmd args] ...
+
+Commands are:
+
+help
+- Prints this message
+
+init [WKSP]
+- Prepare an XDP environment by pinning the XDP redirect
+  program and associated eBPF maps in /sys/fs/bpf/[WKSP].
+  All kernel OS resources allocated by fd_xdp_ctl will be
+  local to their [WKSP], allowing for multiple installations
+  on the same machine.
+- /sys/fs/bpf must be a valid bpffs mount.
+- Creates the following files:
+  /sys/fs/bpf/[WKSP]/udp_dsts  eBPF map of UDP/IP listen addrs
+- Must be re-run after each reboot.
+- Requires CAP_SYS_ADMIN, see capabilities(7).
+
+fini [WKSP]
+- Destroy all kernel resources installed by fd_xdp_ctl app
+  [WKSP] including any XDP programs and eBPF maps.
```

Comment:
```suggestion
- Destroy all kernel resources installed by fd_xdp_ctl in the given
  [WKSP] including any XDP programs and eBPF maps.
```

Reply:
Ooh, that's confusing - I assumed WKSP was a Firedancer workspace.

Reply:
Maybe rename it to something like `BPF_DIR`? Just to be clear it's not a Firedancer shared memory workspace. The `bind-xsk` command is an example of where overloading `WKSP` can be confusing.